### DISCUSSION TITLE
Thinking Cap #4

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#12] [Assignments]
## Prompt:
> What do you think about design justice? How might you apply it in your designs going forward?

### Note: Responses submitted after Sunday 2pm will not be eligible for awards.

This week's thinking cap requires completing the [reading](https://github.com/albertkun/211A-ASIAAM-191A/blob/master/Week_04/Materials/Design_Justice.pdf) beforehand.

## Submission
Make a new comment below in at least 3 - 4 sentences and respond to any two students posts before class begins.

**Comment by edmondywen** [comment]
The most thought-provoking part of the reading to me was the analysis about how design justice is heavily constrained by capitalism’s profit incentive which motivates companies to design products that sell best to the largest market size, which in the US, generally represents white, cis, middle-class males. In spite of this, there have been successful moves to make technology more accessible to marginalized groups. Constanza-Chock highlights how the LGBTQ community and in particular drag queens “successfully [organized]” so that Facebook would modify its ‘real name’ policy which privileged Eurocentric names. This example underscores the importance of constantly unsettling hegemonic spaces by advocating for the rights and needs of everyone. This strategy is a component of the design justice framework which seeks to center the voices of those directly impacted by a product and to work towards non-exploitative solutions that build a sustainable community. In the future, I believe that it is important for me to actively seek out feedback from everyday people so that I can continually adjust my projects to best accommodate for the everyone’s needs.

**Reply by cynthinee** [reply]
Hi Edmond-I agree that regular feedback with community members with whom you work is super important. I think we're seeing a lot of fields finally understand the need to shift from deficit approaches to more asset-based approaches, which is great! But also it's also incredible that this is revolutionary thinking because intuitively this is something that designers should have been employing all along.

**Reply by abi-200** [reply]
I also thought that capitalism's hold on design justice was extremely interesting to her about because while it is obvious that businesses will not prioritize consumer well-being over profits, reading more about the different prioritization of wants versus needs really highlights the ethical dilemma at its core here. I think this easily ties into the idea of social responsibility and societal obligation, and the degree to which individuals feel they have an ethical obligation to others can differ between cultures and borders. By design, some areas will prioritize this less/more than other regions. In addition, while I knew that profit drove most of what developers choose to produce, even if it is only to lean to the desires of a more privileged group of spenders, the point the paper makes about the problems faced by the people without their basic needs met being harder to solve was also very thought-provoking to me. This ties back to societal ethics and obligation, such that those in power have to actively decide to take the harder job and help work step by step to dismantling the more difficult infrastructures, rather than choosing the easier payout.

**Reply by luigimagdamit** [reply]
I think the first point you bring up is very important about design justice's relationship with capitalism. Its important to discuss how the existing societal framework affects the design culture of our culture. I also really admire your last point about being accommodating in your own projects and design, highlighting the importance of feedback from the people and communities around us.

**Comment by abi-200** [comment]
I really enjoyed how this paper framed design as problem-solving, and I learned how design can also incorporate processes and systems in place that seeks to regulate/streamline processes, even in the political and economic sectors - previously, I had only thought of design as physical art design, graphic art, etc. Once the definition was opened up, I could begin to see many more examples of just how design justice has been ignored among major designers, and about how many users do not have adequate/equitable access to designs at hand.
Some examples given in the article of design inequity that most ring true to me are the concepts of immigration status and online Eurocentrism, at both the community and institutional levels . For example, many job search websites and ads post about potential opportunities for students, and yet both me and some friends have faced situations where we read the description, get to the interview stage, and are later told that due to immigration circumstances we are ineligible (and by website design, are unable to filter such opportunities out). My own name is always autocorrected to a more Eurocentric one on Google/Apple, and one of my friends had to use a false last name on Facebook because the brand would not accept her last name as being legitimate. 
For the educational example given, I have previously spent time studying how educational inequities begin when young, as wealthier people in urban neighborhoods move, or put their child in private schools, thereby decreasing enrollment, funding, and PTA participation for neighborhood public schools that are often the only option to low income elementary students and students of color. However, I never considered this a violation of design justice as much as I did an ethical and institutional injustice, so I think opening up my mind to how design is so pervasive for everybody living in society helps solidify the importance of ensuring equity to the best of our ability, at all hierarchical stages
On this same note, I enjoyed how the paper itself talks about how design is a back and forth relationship between us and our planet, and how everybody has a contribution/design should not be limited to specialist design. As the article points out, oftentimes the people negatively impacted by the inadequate design of power structure (directly affected communities) are the ones with the least power to change such structure, and that the contributions of individuals form these communities will provide the most comprehensive and inclusive design that's seeks to absolve particular problems. Another  example in the article that I found very interesting was intersectionality, brought up at the beginning - I knew that intersectional thought was the most thorough and thus likely to be the most accurate when it comes to data collection of personal stories/problems, but I have never had it laid out for me how people go to lengths to establish loopholes based on intersectionality that prevent people from seeking justice by design of the system (ex. Black women workers rights examples listed). To go back to the educational example, I can tie to intersectionality because the solution is a combination of both adding diversity and ensuring adequate social structures and practices that allow people of underrepresented communities to thrive (that is, the solution to fixing lack of diversity is not only to add diversity and leave without ensuring proper systems in place for people to thrive).
In my design going forward, I will likely focus on the individual and community levels mentioned in the paper as they are most prevalent to what I currently have power to change, and in this class, I can make sure that my survey questions have options for everybody to fill out demographically, and will not exclude certain groups. In addition, in the future, depending on what I choose to do as a career/specialize in, I will do my best to get out of the mindset that my word and the words of community members are not on equal footing, and that I am merely the facilitator and nothing more.

**Reply by cynthinee** [reply]
I really resonated with what you recalled about power structures and directly affected communities. In higher education, there are a lot of students who study areas to go into careers with the intentions of combining their lived experiences in an affected community and as a specialist in the field they enter. This is such a great opportunity for various fields to change and grow with new, diverse perspectives, but often the opposite happens. Companies fail to see the incredible affordance of having such individuals working for them, these individuals try to make change regardless, and unfortunately so many get burnt out from toxic work environments. I'm really hoping the toxic workplaces that aren't thinking about things like design justice become obsolete, while those who want to make meaningful change in conversation with communities prevail. One can hope?

**Reply by hrfrahn** [reply]
Your point about how pervasive design is in everyday life really struck me; it's easy to forget, but pretty much everything we interact with in modern society has been intentionally designed in a certain way/for a certain group. This "invisibility" of design makes it easy to ignore injustices in design by pretending that "it's just the way things are" and further underscores the need for true design justice.

**Comment by cynthinee** [comment]
I really appreciate any framework that decenters expertise and actually works with communities. Moreover, placing one or a few self-appointed authoritative figures at the center of design, or research, is counterproductive. I really love the bullet point “Before seeking new design solutions, we look for what is already working at the community level. We honor and uplift traditional, indigenous, and local knowledge and practices.” Communities already know how to do so many things effectively and to leave them out of the design process makes no sense.

**Reply by abi-200** [reply]
That bullet point also resonated with me because it is somehow the simplest step towards obtaining a direct picture from the communities in question, but can also be extremely underused/forgotten. Elitism can be rampant and more prominent the higher up in a hierarchal chain a designer/developer/project manager is, and dismantling this structure by not only increasing community representation, but also by providing adequate tools and training to the participant, is one key way to doing this.

**Reply by hrfrahn** [reply]
You made a great point about the importance of "looking for what's already working" in communities; it seems like people in academic settings (like us in this class!) often assume that whatever fancy new technology/solution that they've come up with is the clear answer to a problem, when in reality there are often real-world solutions that people have been using for years that are just as effective.

**Reply by luigimagdamit** [reply]
I agree very heavily with your last point. I think that uplifting and encouraging existing design solutions from different groups is really the way to go. I feel as though historically, many cultural and scientific advances would not have as fast or radical without the exploration of nontraditional practices.

**Reply by edmondywen** [reply]
I agree that the community point is super important because if the purpose of design is to make a tool accessible/marketable to a group of people, then it makes sense that community based solutions should be prioritized. Unfortunately, many research and market strategies tend to over-prioritize "expertise" over real-world experience

**Reply by ronnie-blanche** [reply]
Hi, Cynthia! I appreciate how you pointed out the counter-productivity that comes with centering design and research around people with authority. Frequently, we put these people on a pedestal because of how they are educated, the research they do, etc. However, education and research are all benefits of this power hierarchy that results from the systems of oppressions we have in place. So by shifting the power back to communities, we help to break the power dynamics that exist and uplift marginalized communities.

**Comment by hrfrahn** [comment]
One part of the reading that really stood out to me was Costanza-Chock's emphasis on "the value of situated knowledge over universalist knowledge". In other words, design justice requires designers to avoid defaulting to overbroad one-fits-all ideas of design. Instead, they should work with the people they are designing for ("those who occupy a subjugated standpoint" in Costanza-Chock's analysis) to ensure that their design works for the community. This idea of "situated knowledge" is readily applicable to the work we're doing in this class; for example, we shouldn't assume that we know the best way to design a survey, and should try to reach out to whichever community we're trying to gain understanding about when designing the survey.

**Reply by edmondywen** [reply]
I love the point about situated knowledge. I believe that designers like myself need to humble ourselves by acknowledging that it is impossible to know everything. Instead, we should celebrate ideas that work and uplift the people behind them. This would bring qualified people into spaces that need them while creating tools that more people can use.

**Reply by emilysuh** [reply]
I appreciate your emphasizing of situated knowledge. A parallel I see here with movement work is that often times localized movement work is far more participatory, democratic and effective than national movement work for the same reason. What works in one community may not work in another because there are important differences in local factors and circumstances.

**Comment by luigimagdamit** [comment]
Great read. I liked the points brought about in the section “Designers: Who gets(paid) to do design?” This section really stuck out to me because I personally detest the notion that design should be dictated by the hands of the experts and the few. I think the inverse belief really resonates with my personal values of democratizing and mobilizing people. 

Also, I think this effect of paid designers gives a really nasty feedback loop to the designs we are exposed to. One such design I can think of is the continued minimization of logo design. I’m not a fan. It’s art created for the sake of generating a marginal profit and making brands more “accessible” which further perpetuates corporate friendly art as the more desirable medium, and disincentivizes paid designers to stray away from this and reduces community exposure to other styles of design. 

I think this point really shows why exploring other ways of knowing is so important. Design from community empowers and allows us to communicate on equal terms.

**Reply by emilysuh** [reply]
I agree with the points you shared, Luigi. Since you brought up logo design, you might find this program called [Give a Brand by Thinkso](https://thinkso.com/giveabrand) interesting. They take nominations every year for an underfunded nonprofit organization and give free design services for the selected organization, including a logo. It's obviously a very individualized solution to the inequity faced by community organizations in having to "brand" in order to solicit funding, but I do know an organization which received this and benefited from it!

**Reply by luigimagdamit** [reply]
Thanks for sharing this! I checked it out and its really nice to see solutions like this that aim to benefit community orgs.  It must be really nice to see organizations around you benefitting from this cause as well.

**Comment by emilysuh** [comment]
I found design justice to be a very helpful framework in thinking about design and equity together. In particular, I enjoyed the critique of professionalization and expert-based models of designing, which have the impact of disempowering community from designing (i.e. people who are not "experts" can't participate in the decision making) and commodifying design knowledge which should be accessible to everyone. In applying design justice to my work going forward, I will deliberate more on how the design can be sustainable over time and help facilitate or give tools to the community that can be used to further the community's goal. As an example, in my work I often rely on public Google documents to share action toolkits (e.g. call your legislator to make community demands with scripts and numbers), and sometimes these documents are word-heavy without many visual design cues to help people quickly filter through and understand the information. This method is also inaccessible to some of our members who may not have computer literacy or reasonable access to Google documents. To remedy this, it may make sense to create a mobile text version of the toolkit, or design a different kind of participatory action so people without computers can also participate (e.g. a print-out to be mailed, or a gathering where people can handwrite letters to the legislator instead of making calls).

**Reply by ramirezirvin** [reply]
I totally agree with you Emily, having multiple forms of participation is a great way to incorporate inclusivity. As we may not know the resources people have on hand to participate. Having a more friendly based document for people of all literacy knowledge can also help people feel more comfortable filing out documents as they are more comfortable understanding what is being asked of them and easily intake the information provided.

**Reply by perezsara59** [reply]
As people who work and produce a lot of things that are more technical that are meant to be viewed on a device, I definitely agree and think its important to think of alternatives. I remember when the news would say to go to this website to make your appointment to get vaccinated, my first thought was "what if they dont have access to the internet or a device and theres a language barrier with their neighbors so they cant ask them for help?" How are these communities being affected, and how are vaccination efforts maybe adjusted to reach these communities?

**Reply by ronnie-blanche** [reply]
Hey, Emily! I really appreciate how you pointed out that some of your colleagues might not have access to computer literacy or Google documents. A large part of justice has to do with does involve making resources more accessible and empowering for marginalized communities. Given that, I think design justice can definitely help in terms of making things more accessible, but also easier to understand/comprehend.

**Comment by ramirezirvin** [comment]
I really enjoyed the reading and how it displays the importance of intersectionality within individuals.
Design justice, if it includes everyone from every part of the spectrum can be quite powerful. It can help provide empowerment in work and media-based communities that lack diversity. Either in the workforce, or media representation, while breaking the eurocentric views that the west has. If they are able to help the communities in need of representation they can provide more awareness of people who lack resources. As for further applications, I think it is best to implicate a safe space for those communities and be aware of the space you are in to make everyone feel included in all fields of life. It is better to be informed of the space you want to create for the people you want to include to expand to an all-inclusive work and provide resources if needed. Not everyone has had the same advantages as everyone else, so providing a judgement-free environment could better the results each individual may bring to the table.

**Reply by perezsara59** [reply]
I agree and I definitely think creating a safe space will help encourage people, especially within marginalized communities, to not only learn skills they feel intimidated or feel like isn't something they can do because they either weren't granted the same opportunity as others to have access to or isn't something they were aware of but also creating a network of people you can relate to and feel safe asking questions or reaching out to for help.

**Reply by eandrusenki** [reply]
I agree that part of design justice should not just be about answering questions about who you are empowering, but that another very important aspect of design justice is creating a safe space and inclusive environment that can also be a way of empowering individuals and communities. This can also be a way of getting more members of the community to get involved, if they feel welcome, rather than environments where two people who believe they known everything are in charge.

**Comment by perezsara59** [comment]
I think design justice is a great way to be more mindful and self-aware of our actions as producers that use/work with design. The article highlights accessibility, reproducing/contributing to white supremacy, heteropatriarchy, capitalism and settler colonialism which isn't something that may cross peoples mind with "good intentions" which the article mentions is not enough. I agree and I think working on things you feel like you have a stake in or is something you can relate to can help not just amplify a relevant story but include the community in the work. There's this documentary I watched recently called _Holding Ground: The Rebirth of Dudley Street_ that highlights community organization and urban planners working for the community as well as teaching people within the community the skills they know. Going forward I can think of all the different communities my designs may affect, who is being limited, who is being harmed and how I can do better whether that be stepping down, giving the opportunity to someone else or sharing my skills with someone so they can empower and amplify their own story. One thing I think I have been conscious about while map making is the color scheme and who I may be leaving out my using certain colors(i.e. red or green). There's a tool called color brewer that recommends color palettes that are colorblind safe which I think is neat because it expands the community that can view and interpret your work.

**Reply by ramirezirvin** [reply]
Thank you for helping me broaden out my views on inclusivity. I am often not aware of people who are color sensitive and to help the community, this is a great thought to think about and incorporate into future projects. This is a very helpful resource and thought to have in mind when working with colors, especially with graphing and mapping.

**Reply by eandrusenki** [reply]
It is extremely important and I like that you brought up that we should participate in projects which we might have a stake in, and that is a way of empowering and amplifying voices. I have a few friends that are colorblind GIS-ers and it is interesting to understand what communities I have been leaving out or exclusively including just by the designs and cartography of my maps.

**Comment by ronnie-blanche** [comment]
Something I really appreciate about Chock's reading is that they their focus on design justice isn't only the community, but marginalized communities specifically! A lot of the time when dealing with marginalized communities, our approaches are often general/cookie cutter rather than personalized and adjusted to reflect the fact that different communities face different obstacles. For example, design justice that is more conducive to helping a First Nation tribe might not be the most effective approach to stopping anti-Asian hate. Thus, I appreciate how Chock proposes a number of ways to enact design justice! That being said though, justice can only really be fully enacted if we get rid of systems of oppressions – the same systems that have influenced how we understand maps and GIS data. And the only way to really abolish these systems is through the power of the people they benefit. Thus, design justice should also focus on bringing awareness to the power disparities and how people can use their power to help put an end to the oppression these communities face.

**Comment by eandrusenki** [comment]
I thought this was a great reading and something I had never really heard of before this class/reading. I think it is really important that we ask ourselves the questions posed in the reading and that we take into consider communities we are helping, especially ones that have been oppressed and those that fall in an intersectionality. I think design justice can be "optimized" when outsiders who claim to know the most are not put in control of design, but rather members of community, with an emphasis on the plural, and that rather than having one or two individuals be in the center, it's a group or smaller community. Another way, as discussed multiple times is that we give such communities and individuals the power and knowledge of design abilities, so they can be empowered and empower themselves and the knowledge they already have.
